### ISSUE TITLE
[KF 1.5] CSI-s3 Storage Integration in kfp-tekton fails to deploy.

### ISSUE THREAD

**Issue by VaishnaviHire** [opendatahub-io/manifests#90]
KFP-Tekton v1.1.0 added support for csi-S3 storage. https://github.com/kubeflow/kfp-tekton/releases/tag/v1.1.0

However, the `kfp-csi-s3-` pods fail to deploy on OpenShift 4.8 and throw the following error:
```
MountVolume.SetUp failed for volume "registration-dir" : hostPath type check failed: /var/data/kubelet/plugins_registry is not a directory
```

**Comment by VaishnaviHire** [comment]
This issue is caused because the default kubelet path in OpenShift is `/var/lib/kubelet` instead of `/var/data/kubelet`.

Immediate fix can be to patch the all instances of `/var/data`  [Daemonset](https://github.com/opendatahub-io/manifests/blob/v1.5-branch-openshift/apps/kfp-tekton/upstream/third-party/kfp-csi-s3/csi-s3-deployment.yaml#L353) to `/var/lib`

However Tekton v1.20  will have a fix to use custom kubelet path - https://github.com/kubeflow/kfp-tekton/pull/900

cc: @LaVLaS

**Comment by VaishnaviHire** [comment]
fixed in #85
